README.md

```diff
@@ -181,6 +182,14 @@ To ensure that Teleporter can be deployed to the same address on every EVM based
 
 `deploy_teleporter.sh` will send the necessary native tokens to the deployer address if it is provided with a private key for an account with sufficient funds. Alternatively, the deployer address can be funded externally. The deployer address for each version can be found by looking up the appropriate version at https://github.com/ava-labs/teleporter/releases and downloading `TeleporterMessenger_Deployer_Address_<VERSION>.txt`.
 
+## Upgradeability
```

Comment:
Lets include something in the "Deploy Teleporter to a Subnet" section about deploying a registry contract if one doesn't already exist on the chain.

From the perspective of a dev trying to get something set up, I mostly just want to know what commands I need to run first, with needing to read as little as possible.

Reply:
Moved the deployment step from `upgrades/README` to the main readme under "Deploy TeleporterRegistry to a subnet's chain". For now using `forge create`, still looking at how to do in a script with bytecode.